lib/constructs/persistence/database.ts

```diff
@@ -45,11 +45,11 @@ export class DynamoDatabase extends cdk.Construct {
         });
 
         this.tables[Collection.FEEDS] = new Table(this, 'dynamodb-feeds-table', {
-            partitionKey: {name: "category", type: AttributeType.STRING},
+            partitionKey: {name: "type", type: AttributeType.STRING},
             billingMode: BillingMode.PROVISIONED,
             encryption: TableEncryption.DEFAULT,
             removalPolicy: cdk.RemovalPolicy.RETAIN,
-            sortKey: {name: "created_at", type: AttributeType.STRING},
+            sortKey: {name: "update_at", type: AttributeType.STRING},
```

Comment:
Same

It's should be created_at anyway. You don't want an updated version of a older post to precede a fresh new post.